# FormationNavigator.msg

float64 count_number
float64 position_x
float64 position_y
float64 position_z

float64 homeLat
float64 homeLon
float64 homeAlt

time gps_time

int32 gap_x
int32 gap_y

float64 leaderAirspeed
float64 leaderHeading
